tools/theme/test/theme-devmode.test.ts

```diff
@@ -40,36 +39,56 @@ describe('Dev mode', () => {
                 level: 'default',
             },
         });
-        consoleWarnStub.restore();
+        consoleWarnStub.reset();
```

Comment:
This implies that there should be an `afterEach()` in this `describe` and maybe an `after` as seen in something like https://github.com/adobe/spectrum-web-components/blob/main/packages/action-button/test/action-button.test.ts#L309-L321

Reply:
Thanks!